app/templates/components/hero-desktop.html

```diff
@@ -0,0 +1,66 @@
+<div
+    class="hero-section hidden fixed top-0 left-0 bg-corporate-blue text-white h-screen max-w-hero overflow-hidden desktop:flex"
+>
+    <div
+        id="slider"
+        class="w-2x-hero h-full translate-x-0 flex ease-in duration-300 {% if false %}-translate-x-1/2{% endif %}"
```

Comment:
The stuff after `{% if false %}` is not used, right? Was it just added for testing/debugging?